pontoon/api/urls.py

```diff
@@ -13,4 +14,32 @@
         r"^graphql/?$",
         GraphQLView.as_view(schema=schema, graphiql=DEV),
     ),
+    # API v1
+    path(
+        "api/v1/",
+        include(
+            [
+                path(
+                    # User actions
+                    "get-user-actions/",
```

Comment:
The HTTP protocol already includes the GET verb.

```suggestion
                    "user-actions/",
```